test/metabase/channel/impl/http_test.clj

```diff
@@ -54,8 +55,11 @@
 
 (defn do-with-server
   [route+handlers f]
+  (def route+handlers route+handlers)
   (let [handler        (->> route+handlers
-                            (map :route)
+                            (mapv :route)
+                            (cons (compojure.route/not-found {:status-code 404 :body "Not found."}))
+                            reverse ;; not found is last
```

Comment:
It was already a vector, why not use `conj` instead of `cons` to add it to the other side?

Reply:
bc this is a thread last 😞

Reply:
How about using as->?